src/main/java/com/github/mob41/blapi/pkt/CmdPacket.java

```diff
@@ -72,95 +72,117 @@
      * @param payload
      *            The data to be sent
      */
-    public CmdPacket(Mac targetMac, int count, byte[] id, byte[] iv, byte[] key, CmdPayload cmdPayload) {
+    public CmdPacket(Mac targetMac, int count, byte[] id, byte[] iv, byte[] key, AES theAes, CmdPayload cmdPayload) {

```

Comment:
TODO: Remove unnecessary variables "iv, key" and rename "theAes" to "aesInstance"

Reply:
Per instance, we should only have one AES object that is referred too. IT only needs to be created on initial, then once again after the auth is called.

Reply:
I mean the iv and key is not used here.

Reply:
Ahh yes, that would be true. Makes Sense to streamline.